scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: ardana-ccp-c0-m1-mgmt
+  gather_facts: False
+  include_vars: "provider.yml"
+  vars:
+    caasp_image_url: "http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+    caasp_image_path: "/tmp/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
+  tasks:
+  - name: Install caasp-openstack-heat-templates
+    always_run: True
+    become: yes
+    zypper:
+      name: caasp-openstack-heat-templates
+      state: present
```

Comment:
I'd suggest to have retries here, as this task relies on the network and is prone to network issues.

For this you should, for example:

register: _install_caasp_heat_templates
until: _install_caasp_heat_templates | success #or is success if that's supported by your ansible version
retries: 5
delay: 2

Reply:
Done

Reply:
why would installing a package rely on network? the package is local. the retries here imho don't make sense, it should just try once and fail if it doesn't work.

Reply:
makes sense, removed it.

Reply:
@dirkmueller isn't the package installed from a repo?
If that's the case, you'll always rely on the network to get your package.